features/org.wso2.carbon.appmgt.core.feature/src/main/resources/jaggery-modules/store/scripts/user.js

```diff
@@ -14,7 +14,7 @@ var user = {};
          * "/" will be replaced.
          */
 
-        return username.replace('@', '-AT-').replace('/', ':');
+        return String(username).replace(/@/g, '-AT-').replace(/\//g, ':');
```

Comment:
add a comment  about what's done here. we can make regex constants if needed to make it meaningful.